Tests/ApolloTests/HTTPTransportTests.swift

```diff
@@ -257,6 +257,8 @@ extension HTTPTransportTests: HTTPNetworkTransportRetryDelegate {
       retryHandler(true)
     case .invalidResponse:
       retryHandler(false)
+    case .persistedQueryNotFound,  .persistedQueryNotSupported:
```

Comment:
mind bumping the 2nd case onto a new line? Makes it easier to tell how many cases are being handled with a single statement.

Reply:
updated